index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>osdc.github.io</title>
+    <link href="https://fonts.googleapis.com/css?family=Assistant:200|Monoton" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:300" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="styles/style.css">
+  </head>
+  <body>
+    <div class="main">
```

Comment:
Too many "divs" - redundant and not SEO friendly. Please try to use HTML5 tags like `<main>`, `<header>`, `<section>` wherever possible.
For example, I guess you could replace `<div class="main">` by simply `<main>` here.